### ISSUE TITLE
test KO after creating the API 

### ISSUE THREAD

**Issue by Nesslax** [diegohaz/rest#240]
I just installed the API and added 1 entity, 
After following the steps in the video, I cant run any thing 
I tried to do npm test 

```
Test Suites: 6 failed, 6 total
Tests:       83 failed, 83 total
Snapshots:   0 total
Time:        28.513s
Ran all test suites.
```

can someone help me please? 

  ```
GET /broadcasts/:id 200

    Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.Error: Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.

      at mapper (node_modules/jest-jasmine2/build/queueRunner.js:27:45)
```

**Comment by diegohaz** [comment]
Are you following the instructions on https://github.com/diegohaz/rest#playing-locally?

**Comment by HugoLiconV** [comment]
> Are you following the instructions on https://github.com/diegohaz/rest#playing-locally?

I did follow the instructions but I'm getting the same error.

So just to clarify, I have mongo running in the background, then in a terminal, I run `npm run dev` and in another one, I run `npm test` and It doesn't work.

I tried to run just `npm test`, without the `npm run dev` and I get the same result.

my node version is 12.4.0

**Comment by I322871** [comment]
Should be fixed with this pr: https://github.com/diegohaz/rest/pull/246

**Comment by HugoLiconV** [comment]
> Should be fixed with this pr: #246

That was fast hahaha.

Thank you so much for that, it solved the problem.
